### PR TITLE
chore: cfnspec bump fails when new modules need to be created

### PR DESCRIPTION
**Pull request by nija-at**
## Commit Message
chore: cfnspec bump fails when new modules need to be created (#6689)

This is because the cfnspec module expects `cfn2ts`, `cdk-build-tools`
and `decdk` to be built when calling 'create-missing-libraries'.

Update 'create-missing-libraries' so it no longer depends on the build
artifacts of `decdk`, as well as, update the bump script to now 'build up'
`cfn2ts`.
## End Commit Message

----

### Testing

```shell
⇒ ./scripts/bump-cfnspec.sh
  ...
  ...
⇒  git show --name-only HEAD
commit 31300a83148283d364608145c742043bc70957ef
Author: Niranjan Jayakar <nija@amazon.com>
Date:   Thu Mar 12 14:46:10 2020 +0000

    feat: cloudformation spec v11.4.0

packages/@aws-cdk/aws-chatbot/.gitignore
packages/@aws-cdk/aws-chatbot/.npmignore
packages/@aws-cdk/aws-chatbot/LICENSE
packages/@aws-cdk/aws-chatbot/NOTICE
packages/@aws-cdk/aws-chatbot/README.md
packages/@aws-cdk/aws-chatbot/lib/index.ts
packages/@aws-cdk/aws-chatbot/package.json
packages/@aws-cdk/aws-chatbot/test/chatbot.test.ts
packages/@aws-cdk/aws-codestarconnections/.gitignore
packages/@aws-cdk/aws-codestarconnections/.npmignore
packages/@aws-cdk/aws-codestarconnections/LICENSE
packages/@aws-cdk/aws-codestarconnections/NOTICE
packages/@aws-cdk/aws-codestarconnections/README.md
packages/@aws-cdk/aws-codestarconnections/lib/index.ts
packages/@aws-cdk/aws-codestarconnections/package.json
packages/@aws-cdk/aws-codestarconnections/test/codestarconnections.test.ts
packages/@aws-cdk/cfnspec/CHANGELOG.md
packages/@aws-cdk/cfnspec/cfn.version
packages/@aws-cdk/cfnspec/spec-source/000_CloudFormationResourceSpecification.json
packages/decdk/package.json
packages/monocdk-experiment/package.json
```

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: scripts/bump-cfnspec.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@ if [ ! -f "lerna.json" ]; then
 fi
 
 ./install.sh
+./build.sh
```
Comment (rix0rrr) [inline scripts/bump-cfnspec.sh]: This is going to take forever!

Other suggestion: do `buildup` from `cfn2ts`? That should include `cfnspec`.

Comment (nija-at) [reply scripts/bump-cfnspec.sh]: That's not going to work. 

Once `cfn2ts` is built; it expects `@aws-cdk/core` to be built (while running a build on the newly created library). Following that it expects [decdk to be built](https://github.com/aws/aws-cdk/blob/bed53578df64b6a77c6a6e434c235ef4aefde300/packages/%40aws-cdk/cfnspec/build-tools/create-missing-libraries.ts#L281).

At this point, aren't we building everything?

Comment (rix0rrr) [reply scripts/bump-cfnspec.sh]: :(

Comment (rix0rrr) [reply scripts/bump-cfnspec.sh]: Hmm I've shipped this but does it even need to do a validating build? The validating build (that builds core and decdk and everything else) will be run as part of the PR build, no?

Only thing we need to do here is make sure that build WOULD have succeded.

Comment (nija-at) [reply scripts/bump-cfnspec.sh]: I've re-done the implementation and addressed your feedback 😊

**Pull request merged**